elements/pfe-styles/src/pfe-base.scss

```diff
@@ -67,24 +65,90 @@ th {
 
 // PatternFly Styles
 
-html {
+:root {
   @include pfe-set-broadcast-context(light);
   font-family: sans-serif;
   line-height: pfe-var(line-height);
+  font-size: pfe-var(FontSize--root);
 }
 
+// leaving this out, because the design guidelines are more nuanced than shrinking everything. this is heavy-handed
```

Comment:
I like the idea of leaving this in but commented.  Can remove the interpolation on `pfe-breakpoint`